packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.props.ts

```diff
@@ -1,6 +1,18 @@
 import * as React from "react";
 import { IHorizontalOverflowClassNameContract, IManagedClasses } from "@microsoft/fast-components-class-name-contracts-base";
 
+/**
+ * Scroll interface for consumers
+ * 'start' is when the horizontal overflow scroll is all the way left in LTR (all the way right in RTL)
+ * 'end' is when the horizontal overflow scroll is all the right in LTR (all the way left in RTL)
+ */
+export interface IScrollChange {
+    start: boolean;
+    end: boolean;
+}
+
+export type onScrollChange = (scrollObject: IScrollChange) => void;
```

Comment:
I believe our naming convention means this should be pascal case.

Reply:
Do you mean `OnScrollChange`?

Reply:
Yep